conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -21515,6 +21515,10 @@ msgid ""
 "#%(r2)d"
 msgstr ""
 
+#: lms/templates/instructor/instructor_dashboard_2/data_download.html
+msgid "OK"
```

Comment:
The _OK_ string is already in this file - see [here](https://github.com/proversity-org/edx-platform/blob/38a1201998fcf7341f9f841a29c0ab110506aa18/conf/locale/en/LC_MESSAGES/django.po#L16379). Could you please remove the section you added in this PR and add:
` #: lms/templates/instructor/instructor_dashboard_2/data_download.html` above [this line](https://github.com/proversity-org/edx-platform/blob/38a1201998fcf7341f9f841a29c0ab110506aa18/conf/locale/en/LC_MESSAGES/django.po#L16379). Thanks!

Reply:
done